nameko/events.py

```diff
@@ -183,10 +168,69 @@ def __init__(self, source_service, event_type, handler_type=SERVICE_POOL,
         super(EventHandler, self).__init__(
             queue=None, requeue_on_error=requeue_on_error)
 
+    @property
```

Comment:
I don't think attribute access implies trivial. `property` gets used all the time to hide lazy creation of expensive objects, or provide a helpful interface on top of a moderately complex operation (e.g. https://github.com/onefinestay/nameko/blob/master/nameko/web/server.py#L64)

`obj.foo` is just a little bit cleaner than ``obj.get_foo()``